docs/UserGuide.adoc

```diff
@@ -219,22 +230,38 @@ Medicines with quanities of less than or equal to 20 will be displayed in the wa
 
 === Exporting as CSV file format : `export`
 
-You can export the current Medicine Inventory data shown in the GUI as CSV file format. +
+You can export the current Medicine Inventory data shown in the GUI as Comma-separated values (CSV) file format which is supported by many commonly used spreadsheet applications such as Microsoft Excel (Windows), LibreOffice (Linux) or Numbers (MacOS).
+
+This feature is especially useful if you want to save the current Medicine Inventory data in a file so that it can be printed later or if you want to organise the Medicine Inventory data using a spreadsheet application for you, your superior or your supplier's reference.
+
+In addition, you can even use the spreadsheet application to convert the CSV file to Microsoft Excel format and use all the functionalities provided by Microsoft Excel such as drawing graphs or chart to display the overview of your clinic's medicine inventory. +
 Format: `export [FILE_NAME]`
 
 ****
-* If no file name is specified, the export will just name the file according to the date and time of export.
+* The `FILE_NAME` field is optional. You can refer to <<Appendix A: File Naming Convention>> for supported file name format if you want to specify a file name for the exported CSV file.
+* If no file name is specified, the export command uses the default file name format according to the date and time of export.
+* The specific data and time format used if no file name is specified is `dd_MMM_yyyy_HH_mm_ss` e.g. `18_Mar_2019_10_28_00`
+* The file extension for a CSV file is `.csv`
+* The exported file can be found in the default `exported` folder which can be found in the home folder used for your MediTabs application.
+* If the default `exported` folder is not found in the home folder, MediTabs will automatically create the folder when the export command is executed.
 ****
 
 Examples:
 
```

Comment:
missing prefix?

Reply:
My source code implementation for export command since milestone v1.1 did not require f/ prefix since it only requires one optional parameter I do not think it is necessary